backend/src/test/java/com/woowacourse/momo/acceptance/auth/AuthAcceptanceTest.java

```diff
@@ -25,6 +26,20 @@ void login() {
         AuthRestHandler.회원가입을_한다(MEMBER_FIXTURE);
 
         AuthRestHandler.로그인을_한다(MEMBER_FIXTURE)
+                .statusCode(HttpStatus.OK.value())
+                .body("accessToken", Matchers.notNullValue())
+                .body("refreshToken", Matchers.notNullValue());
+    }
+
+    @DisplayName("리프레시토큰을 통해 새로운 엑세스 토큰을 발급받는다")
+    @Test
+    void reissueAccessToken() {
+        AuthRestHandler.회원가입을_한다(MEMBER_FIXTURE);
```

Comment:
```suggestion
        회원가입을_한다(MEMBER_FIXTURE);
```
static 으로 가져와도 될것같아요~

Reply:
전부다 static으로 가져오도록 수정했습니다!